public/video-ui/styles/layout/_topbar.scss

```diff
@@ -3,6 +3,16 @@ $topbarHeight: 50px;
 .topbar {
   background: $color700Grey;
   border-bottom: 1px solid $greyBorderColor;
+
+  &--training-mode {
+    background: repeating-linear-gradient(
+        -45deg,
+        $color500Grey,
+        $color500Grey 10px,
+        $color800Grey 10px,
+        $color800Grey 20px
```

Comment:
Unfortunately opacity hits the all the elements 😢 . I've forced the background colour back instead:

![image](https://user-images.githubusercontent.com/836140/29214555-b4ec8676-7e9f-11e7-94e5-0797e3a05aa6.png)